src/super_gradients/training/transforms/pipeline_adaptors.py

```diff
@@ -83,6 +135,34 @@ def post_transforms_processing(self, sample):
             sample = SegmentationSample(image=Image.fromarray(sample["image"]), mask=Image.fromarray(sample["mask"]))
         elif self.sample_type == SampleType.DEPTH_ESTIMATION:
             sample = DepthEstimationSample(image=sample["image"], depth_map=sample["mask"])
+        elif self.sample_type == SampleType.POSE_ESTIMATION:
+
+            if len(sample["bboxes"]) == 0:
+                sample["bboxes"] = np.zeros((0, 4))
+            if len(sample["is_crowd"]) == 0:
+                sample["is_crowd"] = np.zeros((0))
+
+            bboxes_xywh = xyxy_to_xywh(bboxes=np.array(sample["bboxes"]), image_shape=sample["image"].shape)
+
+            # Update value of keypoints that are not inside the image anymore.
+            h, w = sample["image"].shape[1], sample["image"].shape[0]
```

Comment:
I think the h and w here may be put in a wrong order. If we assume image shape is HWC, then shape[1] is width and shape[0] is height.

Reply:
Oh yeah, I don't know why it's reversed I didnt see, thanks!